### DISCUSSION TITLE
Dashboard roles and permission

### DISCUSSION THREAD

**Discussion by dev-dhull** [filamentphp/filament#3409] [Help]
how to resolve this issue -> when I create filament-shield roles and permissions, my other resource is hidden automatically on the dashboard.

**Comment by danharrin** [comment]
Open an discussion in the plugin repo, not here

**Reply by dev-dhull** [reply]
But I don't know how I can reach the discussion repo of plugin

**Reply by danharrin** [reply]
https://github.com/bezhanSalleh/filament-shield/issues/new
